app/uk/gov/hmrc/agentuserclientdetails/connectors/EnrolmentStoreProxyConnector.scala

```diff
@@ -51,6 +51,18 @@ trait EnrolmentStoreProxyConnector {
     groupId: String
   )(implicit hc: HeaderCarrier, executionContext: ExecutionContext): Future[Seq[Enrolment]]
 
+  // ES11 - Assign enrolment to user
+  def assignEnrolment(userId: String, enrolmentKey: String)(implicit
```

Comment:
Instead of returning `Future[Unit]`, can we return some `Future[StatusValue]`?

Reply:
Everywhere else in the connector the pattern is to return a successful future only in case of success and a failure with a suitable exception otherwise, so I'd like to keep it consistent.
Given that ES11 (and also ES12) doesn't return any content in case of success, I don't know what else to return but Unit